discovery-provider/src/models.py

```diff
@@ -309,3 +309,13 @@ def __repr__(self):
 save_type={self.save_type},\
 is_current={self.is_current},\
 is_delete={self.is_delete}>"
+
+class Credit(Base):
+    __tablename__ = "credits"
+
+    parent_track_id = Column(Integer, nullable=False, primary_key=True)
```

Comment:
Should this match the primary key constraint for `Credit` in the migration?

Reply:
absolutely should. thx -- forgot to change it here